fixtures/analysis.py

```diff
@@ -0,0 +1,9 @@
+import json
+import pytest
+
+
+@pytest.fixture(scope="session")
+def json_analysis():
```

Comment:
This method can be renamed to "getAnalysisData"

and you can put a comment inside 
//parsing analysis json to get data

Reply:
I'm not sure about renaming this methdod to include a verb.

This function won't be called explicitly, instead, it will be pased as a parameter to tests (like [here](https://github.com/windup/windup-cli-tests/pull/1/files#diff-6a31938977d3f2aab6acaa5455e7842ce918579c377e55b9d6e97533a0f86905R6)) So changing its name to include a verb could cause conffusion because people might think the argument is a reference to call the function instead of the result itself.

Reply:
Renamed to `analysis_data`